etc/systemd/system/sidekiq-prometheus-exporter.service

```diff
@@ -3,10 +3,48 @@ Description=Sidekiq Prometheus Exporter
 After=network.target
 
 [Service]
+User=mastodon
 WorkingDirectory=/var/lib/sidekiq-prometheus-exporter
-ExecStart=bundle exec rackup -p31337 -o0.0.0.0
+Environment="PUMA_ENV=production"
+Environment="BIND=0.0.0.0"
+Environment="PORT=31337"
+ExecStart=/home/mastodon/.rbenv/shims/bundle exec puma
```

Comment:
This ensures we're using the same ruby version as what we're using with Mastodon.